cfstart.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+# set -o nounset # This option conflicts with the use of regex matching and $BASH_REMATCH
+
+# At this point we expect that you've already done:
+#   cf set-env <appname> DS_RO_USER <datastore_username>
+#   cf set-env <appname> SOLR_URL <solr_url>
+
+
+mkdir -p /var/tmp/ckan/dynamic_menu
+touch /var/tmp/ckan/dynamic_menu/menu.json
+
+CKAN_INI=config/environments/prod/production.ini
+
+# We need to know the application name ...
+
+APP_NAME=$(echo $VCAP_APPLICATION | jq -r '.application_name')
+
+# ... from which we can guess the service names
+
+SVC_DATABASE="${APP_NAME}-db"
+SVC_REDIS="${APP_NAME}-redis"
+
+# Grab database url from the VCAP_SERVICES env var provided by the platform
+
+DATABASE_URL=$(echo $VCAP_SERVICES | jq -r --arg SVC_DATABASE $SVC_DATABASE '.["aws-rds"][] | select(.name == $SVC_DATABASE) | .credentials.uri')
```

Comment:
If there's only one database bound to the app, then [the `$DATABASE_URL` variable is already set](https://docs.cloudfoundry.org/devguide/deploy-apps/environment-variable.html#DATABASE-URL) so you don't need this line.